packages/perseus/src/widgets/__tests__/radio/base-radio_test.jsx

```diff
@@ -79,4 +79,87 @@ describe("base-radio", () => {
             });
         });
     });
+
+    describe("selecting and deselecting options", () => {
+        it("deselects multi-select choices", () => {
+            // Arrange
+            const apiOptions: APIOptions = {
+                styling: {radioStyleVersion: "final"},
```

Comment:
Since you touched it, it might be a good opportunity to pull out `radioStyleVersion` stuff. From the code:

```
            // TODO(emily): Remove this by Aug 1, 2017, at which point all
            //   callsites should have been switched to using the "final"
            //   designs.
            radioStyleVersion: PropTypes.oneOf(["intermediate", "final"]),
```

Reply:
Jeanette suggested putting this into its own ticket, so will do this separately.

Reply:
Here's the new ticket: https://khanacademy.atlassian.net/browse/LP-12680